generate.xml

```diff
@@ -1906,24 +1906,22 @@
       <package label="test"            name="boost_unit_test_framework-vc141" version="1.64.0.0" />
     </nuget>
     <nuget build_name="secp256k1" compiler="vc120" >
-      <package label="secp256k1"       name="secp256k1_vc120"                 version="0.1.0.14" />
+      <package label="secp256k1"       name="secp256k1_vc120"                 version="0.1.0.16" />
     </nuget>
     <nuget build_name="secp256k1" compiler="vc140" >
-      <package label="secp256k1"       name="secp256k1_vc140"                 version="0.1.0.14" />
+      <package label="secp256k1"       name="secp256k1_vc140"                 version="0.1.0.15" />
```

Comment:
This is the correct package, but from another owner: https://www.nuget.org/packages/secp256k1_vc140/ I've asked him if he will remove so I can continue to publish the full series.